packaging/imagemode/Containerfile.repobase

```diff
@@ -9,6 +9,8 @@ FROM ${BASE_IMAGE_URL}:${BASE_IMAGE_TAG}
 
 ARG DNF_OPTIONS
 
+# Do not run 'dnf upgrade' command to avoid overrides of the base
+# images specified by the container image build pipelines.
```

Comment:
Is this the best advice we can give? Are those images rebuilt daily so we don't need to upgrade the packages?

Reply:
It goes back to the comment in the beginning of this Containerfile. It's crafted for ART pipeline requirements.

Reply:
Ack, I didn't see the comment at the top of the file, so I assumed this is an example for the user (just like the blueprint example in release-info rpm)